internal/handler/msg_aggregate.go

```diff
@@ -248,10 +248,21 @@ func (h *Handler) MsgAggregate(ctx context.Context, msg *wire.OpMsg) (*wire.OpMs
 	}
 
 	cancel := func() {}
```

Comment:
Maybe you can remove this line and/or make it `var cancel func()` for the sake of brevity.

Reply:
The func's default value is nil, which cannot be called